src/main/java/pro/quicksense/modules/common/Result.java

```diff
@@ -0,0 +1,57 @@
+package pro.quicksense.modules.common;
+
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+public class Result<T> implements Serializable {
+    private Integer code;
+    private String msg;
+    private T data;
+
+    public static <T> Result<T> success() {
+        Result<T> result = new Result<>();
+        result.setCode(200);
+        result.setMsg("Success");
+        return result;
+    }
+
+    public static <T> Result<T> success(String msg) {
+        Result<T> result = new Result<>();
+        result.setCode(200);
+        result.setMsg(msg);
+        return result;
+    }
+
+    public static <T> Result<T> success(String msg, T data) {
+        Result<T> result = new Result<>();
+        result.setCode(200);
+        result.setMsg(msg);
+        result.setData(data);
+        return result;
+    }
+
+    public static <T> Result<T> success(T data) {
+        Result<T> result = new Result<>();
+        result.setCode(200);
+        result.setMsg("Success");
+        result.setData(data);
+        return result;
+    }
```

Comment:
```suggestion
    public static <T> Result<T> success(String msg) {
        Result<T> result = new Result<>();
        result.setCode(200);
        result.setMsg(msg);
        return result;
    }

    public static <T> Result<T> success(String msg, T data) {
        Result<T> result = new Result<>();
        result.setCode(200);
        result.setMsg(msg);
        result.setData(data);
        return result;
    }
```
Simplify success method, all success and error response should contain message.